CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,24 @@
-# Code of Conduct
+# Conda Organization Code of Conduct
 
-This repo adheres to the [NumFOCUS Code of Conduct](https://www.numfocus.org/code-of-conduct).
+> **Note**
+> Below is the short version of our CoC, see the long version [here](https://github.com/conda-incubator/governance/blob/main/CODE_OF_CONDUCT.md).
+
+# The Short Version
+
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for the conda Organization.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+
+The conda Organization is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+## Report an Incident
+
+* Report a code of conduct incident [using a form](https://form.jotform.com/221527028480048).
+* Report a code of conduct incident via email: [conduct@conda.org](mailto:conduct@conda.org).
+* Contact [an individual committee member](#committee-membership) or [CoC event representative](#coc-representatives) to report an incident in confidence.
+
+&nbsp;
+
+See the longer version of the Code of Conduct [here](https://github.com/conda-incubator/governance/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
@kenodegard Thoughts on still keeping this at the bottom? Probably redundant?

Reply:
Yeah no need to keep this at the bottom